woocommerce/woocommerce-gutenberg-products-block
# Mini Cart block: re-enable background support

This PR re-enables background support for Mini Cart Block. The background support has been removed by #5477 

### Testing

### Manual Testing

How to test the changes in this PR:

Check out this branch:

1. Install and enable `Gutenberg` plugin.
2. Install and enable `TT1 Blocks` theme.
3. Add the `Mini Cart Block` on a post/page.
4. Verify you can change the background color.
5. Save.
6. Go on the page and check if there are changes.
